node/onbuild/Dockerfile

```diff
@@ -6,7 +6,7 @@ WORKDIR /share
 ONBUILD ARG NODE_ENV
 ONBUILD ENV NODE_ENV $NODE_ENV
 ONBUILD COPY package.json /share/
-ONBUILD RUN npm install
 ONBUILD COPY . /share
```

Comment:
I believe line 8 and 9 are redundant. You could remove line 8 and get the same result. I'm not even sure that the order needs to be reversed - could you just use `ADD` instead of `COPY`? 

I mean this works so don't necessarily want to spend more time on this, just though worth bringing up.